src/plugins/python/pythonbind.cpp

```diff
@@ -470,7 +477,7 @@ _wrap_PyQPoint_setY(PyQPoint *self, PyObject *args, PyObject *kwargs)
 
 
 PyObject *
-_wrap_PyQPoint_x(PyQPoint *self, PyObject *PYBINDGEN_UNUSED(_args), PyObject *PYBINDGEN_UNUSED(_kwargs))
+_wrap_PyQPoint_x(PyQPoint *self)
```

Comment:
Where do these changes come from? This file is normally generated using pybindgen, by executing `python tiledbinding.py`. When I do this with the latest version of pybindgen 0.22.1, it does not make these changes.

Reply:
You are right, sorry about it.  (I used the pybindgen shipped by Debian that's is 0.20.) 

Re-running it with the latest version.